src/main/java/com/catapi/entity/CatFact.java

```diff
@@ -26,4 +28,7 @@ public class CatFact {
     @Column(name = "active_state")
     @Enumerated(EnumType.STRING)
     private ActiveState activeState;
+
+    @OneToMany(mappedBy = "catFact", fetch = FetchType.LAZY)
+    private List<CatFactTranslation> catFactTranslations;
```

Comment:
this is back reference link which is provided by Hibernate. We can get all of translations related to one cat fact